docs/pages/docs/guides/continuous-integration.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Continuous Integration
+description: Recipes for using Turborepo with GitHub Actions, CircleCI, and other continuous integration providers.
+---
+
+# Continuous Integration
+
+Turborepo not only speeds up builds, but also your CI pipeline. Below are a few ways to use Turborepo with various Continuous Integration providers.
+
+## GitHub Actions
+
+The following example shows how to use Turborepo with GitHub Actions and pnpm.
+
+```yaml
+name: CI
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types: [opened, synchronize]
+
+build:
+    name: Build and Test
+    timeout-minutes: 15
+    runs-on: ${{ matrix.os }}
+    # To use Remote Caching, uncomment the next lines and follow the steps below.
+    # env:
+    #  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+    #  TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 6.32.2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: pnpm
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Test
+        run: pnpm turbo run test
+```
+
+#### Remote Caching
+
+To use Remote Caching with GitHub Actions, add the following environment variables to your GitHub Actions workflow
+and make them available to your `turbo` commands.
+
+- `TURBO_TOKEN` - The Bearer token to access the Remote Cache
+- `TURBO_TEAM` - The team to which the monorepo belongs
+
+To use Vercel Remote Caching, you can get the value of these variables in a few steps:
+
+1. Create a Scoped Access Token to your account in the Vercel Dashboard https://vercel.com/account/tokens
```

Comment:
I think we can add the link here: [Vercel Dashboard](https://vercel.com/account/tokens)